Open Banking Sample README changes

mjang opened the pull request:
Hi @qcastel , I did a rough edit of your README -- I think it'll improve how people understand our Open Banking Sample. 

As best I can tell, you didn't change some things after moving this repo from Stash to GitHub.

mjang commented:
If you like @qcastel , I'll open an issue. Also, I don't think I have privileges to ask for a reviewer.

qcastel commented on README.md:
```diff
@@ -15,7 +15,7 @@
 # Open Banking Reference Implementation Sample
 
-Code sample for helping TPPs and ASPSP using the ForgeRock Open Banking reference implementation platform.
+Code sample for helping Third Party Protection (TPPs) and Account Servicing Payment Service Provider (ASPSP) using the ForgeRock Open Banking reference implementation platform.
```
Third Party Provider

qcastel commented:
Thanks @mjang, indeed, the project has been moved from stash to github. No need to maintain the stash one anymore.

A few comments but other than that, looks good!
Thanks!

mjang commented on README.md:
```diff
@@ -25,41 +25,40 @@ License URL	: http://forgerock.org/cddlv1-0/
-We implemented a script which will allow you to generate keys, ready to go, that you can import to Open Banking directory.
-What you need to do:
+We've included a script which will allow you to generate keys that you can import to Open Banking directory.
+To proceed:
```
I think this section needs additional clarification, but I haven't taken the time to analyze the steps in this section.

mjang commented:
Hi @qcastel I've addressed your comments. I don't think I can merge, as I don't have write access to this repo. You're welcome to merge, or to give me write access.
